hack/extract_manifests_images.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# The script extracts all images from specific manifests sub-directories.
+# Future release process enhancements may include an automatic image inventory scan.
+# The reported image list can later be used for image vulnerability scanning and managing license risks management
+
+#VERSION=1.7.0
+output_file="../docs/kf${VERSION}_images.txt"
+
+rm -f .tmp
+# Iterate over all files with names: 'kustomization.yaml', 'kustomization.yml', 'Kustomization' found recursively in the provided list of directories
+for F in $(find ../apps ../common ../example ../contrib/metacontroller ../contrib/seldon \( -name kustomization.yaml   -o -name kustomization.yml -o -name Kustomization \)); do
```

Comment:
Can you also add [/contrib/bentoml](https://github.com/kubeflow/manifests/tree/master/contrib/bentoml)? Was missed in the original list as it didn't exist when https://github.com/kubeflow/manifests/issues/2311 was created.

Reply:
done